build.sbt

```diff
@@ -21,7 +21,6 @@ lazy val root = (project in file("."))
       "io.circe" %% "circe-generic" % CirceVersion,
       "org.http4s" %% "http4s-dsl" % Http4sVersion,
       "org.specs2" %% "specs2-core" % Specs2Version % "test",
-      "org.reactormonk" %% "cryptobits" % "1.2",
```

Comment:
Yes, indeed it was used in the authentication example provided by http4s but I am no longer using it. Thanks.